decidim-core/app/packs/src/decidim/redesigned_heading_tag.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Ensures the aside heading tag is transformed to h2 or h1 if the drawer is shown or hidden
```

Comment:
No, this does not work and should not be implemented like this.

The `<h1>` needs to be the first heading on the page always.